django_tidb/features.py

```diff
@@ -95,7 +95,7 @@ def django_test_skips(self):
                 "check_framework.test_database.DatabaseCheckTests.test_mysql_strict_mode",
                 # Unsupported add column and foreign key in single statement
                 "indexes.tests.SchemaIndexesMySQLTests.test_no_index_for_foreignkey",
-                # https://github.com/pingcap/django-tidb/issues/46
+                # TiDB does not support `JSON` format for `EXPLAIN ANALYZE`
```

Comment:
I assume `EXPLAIN FORMAT = "tidb_json" ...` isn't helping here?

https://docs.pingcap.com/tidb/dev/sql-statement-explain#explain-output-format

Reply:
We can create an alias `TIDB_JSON` => `JSON`, similar to how `ROW` => `TEXT` in https://github.com/pingcap/django-tidb/blob/main/django_tidb/operations.py#L22

But it says that the format and fields of TIDB_JSON are different from the JSON output in MySQL, which is why I didn't do it.